core/src/main/java/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractor.kt

```diff
@@ -111,7 +111,7 @@ fun run(
     // Compute stops
     val stops = ArrayList<ResultStops>()
     for (stop in schedule.stops) {
-        val stopTime = envelopeWithStops.interpolateTotalTime(stop.position) - stop.duration
+        val stopTime = envelopeWithStops.interpolateDepartureFrom(stop.position) - stop.duration
```

Comment:
The `DepartureFrom` version matches the previous implementation, but the intent is `ArrivalAt` (which is why the stop duration was subtracted)

```suggestion
        val stopTime = envelopeWithStops.interpolateArrivalAt(stop.position)
```

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/7609/commits/b7f044ebd08664f02948f7aafb19982fee469f65